osu.Game/Graphics/Backgrounds/TrianglesV2.cs

```diff
@@ -71,7 +73,14 @@ public TrianglesV2(int? seed = null)
         private void load(ShaderManager shaders, IRenderer renderer)
         {
             shader = shaders.Load(VertexShaderDescriptor.TEXTURE_2, "TriangleBorder");
-            texture = renderer.WhitePixel;
+
+            // HACK: we want to use white pixel as a texture here, but it doesn't work with custom texture coordinates for some reason
+            texture = renderer.CreateTexture(1, 1, true);
+
+            var image = new Image<Rgba32>(1, 1);
+            image[0, 0] = new Rgba32(1f, 1f, 1f);
+
+            texture.SetData(new TextureUpload(image));
```

Comment:
Basically I expect `WhitePixel` to work here (as any other texture), but it doesn't.
If it is not worth the time looking into - it's probably better to use any small existing texture instead of creating a new one per drawable.
_Or may be I'm just creating problems out of thin air and this pr is kinda pointless as a whole_